taskmanager/interface.go

```diff
@@ -6,29 +6,15 @@ import (
 	taskq "github.com/vmihailenco/taskq/v3"
 )
 
-// TaskService is the task related methods
-type TaskService interface {
-	RegisterTask(task *Task) error
+// Tasker is the taskmanager client interface
+type Tasker interface {
```

Comment:
Maybe better name would be `TaskEngine` or `Engine` or `Manager`

Reply:
I renamed to TaskEngine. Generally, do we have a naming convention for Interfaces? Like `I`Task, Task`er`, Task`Interface`? In [EffectiveGo](https://go.dev/doc/effective_go#interface-names) they mansion the one-method interfaces only and they propose `-er` suffix.

Reply:
No we don't have such a convention (at least I don't know about it). I suggested other names because as you said `-er` suffix is rather for single method interfaces with the similar name of the method.